core/templates/dev/head/domain/story/EditableStoryBackendApiServiceSpec.ts

```diff
@@ -86,7 +86,7 @@ describe('Editable story backend API service', function() {
       var failHandler = jasmine.createSpy('fail');
 
       $httpBackend.expect(
-        'GET', '/story_editor_handler/data/topicId/storyId').respond(
+        'GET', '/story_editor_handler/data/storyId/topicId').respond(
```

Comment:
I think this should be reversed -- topicId should come before storyId. That also makes more sense from a REST-convention addressing standpoint.

Reply:
Done, removed topicId itself from the story editor URLs.